ansible/roles/brozzler-worker/tasks/main.yml

```diff
@@ -1,24 +1,15 @@
 ---
-- name: ensure canonical partner repo is in apt sources.list
-  apt_repository: repo='deb http://archive.canonical.com/ubuntu trusty partner'
-                  state=present
-  become: true
-
-- apt: update_cache=yes
-  become: true
-
 - name: ensure required packages are installed
   become: true
   apt: name={{item}} state=present
   with_items:
-  - chromium-browser
-  - vnc4server
-  - libjpeg-turbo8-dev
+  - chromium
+  - tigervnc-standalone-server
```

Comment:
`vnc4server` has a proprietary license so I switched over to `tigervnc` which is in the main repositories.